src/api/emailApi.ts

```diff
@@ -36,3 +39,8 @@ export const checkSubscriptionEmail = (data: CheckSubscriptionPayload) => {
     data,
   });
 };
+
+export const verifyEmail = (data: VerifyEmailPayload) =>
+  api.post(`/verify_email/`, {
+    ...data,
```

Comment:
maybe
```
  api.post("/verify_email/", data)
```
instead of 
```
  api.post("/verify_email/", { ...data })
```
?

Reply:
not is correct

Reply:
@marek-siemieniuk-morawski is right {...data} does nothing